app/serializers/rest/instance_serializer.rb

```diff
@@ -24,6 +24,12 @@ def version
     Mastodon::Version.to_s
   end
 
+  def stats
+    { user_count:   User.confirmed.count,
```

Comment:
```ruby
  # ...

  private

  def instance_presenter
    @instance_presenter ||= InstancePresenter.new
  end

  # ...
```

It would be good to create a private method that returns `InstancePresenter`.